### PR TITLE
DML.sql

### PR DESCRIPTION
**Pull request by diyorbekaxmatov**

### CODE REVIEW

File: sql/DML.sql [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    (15, 'Amazing views!', 4),
+    (16, 'Clean and comfortable!', 5),
+    (17, 'Great location!', 4),
+    (18, 'Top-notch service!', 5),
+    (19, 'Impressive amenities!', 4),
+    (20, 'Memorable experience!', 5);
+  
+/* test
+SELECT * FROM Review
+*/
```
Comment (SharifovDeveloper) [inline sql/DML.sql:225]: All very Good Diyorbek

Comment (diyorbekaxmatov) [reply sql/DML.sql:225]: Thank you.

Comment (diyorbekaxmatov) [reply sql/DML.sql:225]: Thank you.

File: sql/DML.sql [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    (5, 301, 3),
+    (6, 302, 3),
+    (7, 401, 4),
+    (8, 402, 4),
+    (9, 501, 5),
+    (10, 502, 5),
+    (11, 601, 6),
+    (12, 602, 6),
+	(13, 702, 7),
+    (14, 801, 8),
```
Comment (SharifovDeveloper) [inline sql/DML.sql:132]: Pay attention to the correctness of the lines

**Review by SharifovDeveloper — CHANGES_REQUESTED** [review]

File: sql/DML.sql (hunk 2 above)
Comment (diyorbekaxmatov) [reply sql/DML.sql:132]: Okay

File: sql/DML.sql [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    (6, 6, 6, '2023-10-15', '2023-10-19', 800.00),
+    (7, 7, 7, '2023-10-28', '2023-11-01', 850.00),
+    (8, 8, 8, '2023-11-01', '2023-11-05', 750.00),
+    (9, 9, 9, '2023-11-10', '2023-11-15', 920.00),
+    (10, 10, 10, '2023-11-05', '2023-11-09', 780.00),
+    (11, 11, 11, '2023-11-20', '2023-11-25', 1050.00),
+    (12, 12, 12, '2023-11-15', '2023-11-19', 880.00),
+	(13, 13, 13,'2023-11-5', '2023-11-9', 889.00),
+	(14, 13, 13, '2023-11-28', '2023-12-03', 960.00),
+    (15, 14, 14, '2023-12-01', '2023-12-05', 880.00),
```
Comment (SharifovDeveloper) [inline sql/DML.sql:188]: Oh Again, This is a Mistake !

**Review by SharifovDeveloper — APPROVED** [review]

File: sql/DML.sql (hunk 3 above)
Comment (diyorbekaxmatov) [reply sql/DML.sql:188]: Sorry!

**Comment by Mirazyzz** [comment]
Looks good.

**Pull request merged**